src/win/core.c

```diff
@@ -80,6 +81,94 @@ static void uv__crt_invalid_parameter_handler(const wchar_t* expression,
 }
 #endif
 
+static uv_loop_t** uv__loops;
+static int uv__loops_size;
+static int uv__loops_capacity;
+#define UV__LOOPS_CHUNK_SIZE 8
+static uv_mutex_t uv__loops_lock;
+
+static void uv__loops_init() {
+  uv_mutex_init(&uv__loops_lock);
+  uv__loops = uv__malloc(sizeof(uv_loop_t*) * UV__LOOPS_CHUNK_SIZE);
```

Comment:
I think it would be nice to either zero out or write some poisoned value on each element of the array, otherwise the remove routine moves garbage around.